packages/mu-transport/index.js

```diff
@@ -39,17 +40,47 @@ module.exports = function transport (createDriver, mu, opts) {
     tearDown: driver.tearDown
   }
 
+
+
+  function constructFailResponse (err, msg) {
+    var pkt = { err: err,
+                response: {},
+                protocol: {}}
+    var elt = muid
+
+    /* istanbul ignore else  */
```

Comment:
not keen on artificially maintaining coverage

question is, if we have to do this, maybe the checks aren't necessary

if they are, then we should test (and handle) the alternatives